synth/src/cli/stdf.rs

```diff
@@ -22,7 +23,7 @@ impl ExportStrategy for StdoutExportStrategy {
     fn export(self, params: ExportParams) -> Result<()> {
```

Comment:
Perhaps we also should rename the `StdoutExportStrategy` to `JsonExportStrategy`?